### ISSUE TITLE
Highlight method is not working for radio button

### ISSUE THREAD

**Issue by Debashis9012** [getgauge/taiko#320]
**Expected behavior**
Highlight method should work for radiobutton selector.

**Actual behavior**
Highlight method is not working for radio button.

**Steps to replicate**
* Create a HTML file
```
<form action="#">
  <p>
    <label>
      <input name="group1" type="radio" checked />
      <span>Red</span>
    </label>
  </p>
  <p>
    <label>
      <input name="group1" type="radio" />
      <span>Yellow</span>
    </label>
  </p>
  <p>
    <label>
      <input class="with-gap" name="group1" type="radio"  />
      <span>Green</span>
    </label>
  </p>
  <p>
    <label>
      <input name="group1" type="radio" disabled="disabled" />
      <span>Brown</span>
    </label>
  </p>
</form>
```
**Open the taiko in REPL mode**

> openBrowser()
 ✔ Browser opened
> goto("file:///.../test.html")
 ✔ Navigated to url "file:///.../test.html"
> highlight(radioButton('Yellow'))
 ✘ Error: Radio Button with label Yellow  not found, run `.trace` for more info.
> 

**Version**
```
commit ffd273be919ad9a650f6e7d265370143f0f11a6f
```

**Comment by NivedhaSenthil** [comment]
Related to #319. Should get fixed when text selection is supported for radiobutton

**Comment by sguptatw** [comment]
@NivedhaSenthil This issue exists even if label for is used as suggested in #319. So it may not be related.

**Steps to replicate**
* Create html file
```
<html>
    <body>
        <form action="#">
            <label for="Red">Red</label>
            <input type="radio" name="gender" id="Red" value="Red"><br>
            <label for="Yellow">Yellow</label>
            <input type="radio" name="gender" id="Yellow" value="Yellow"><br>

        </form>
    </body>
</html>
```
* Open taiko in REPL mode
```
> openBrowser()
 ✔ Browser opened
> goto("file:///.../HTMLElements.html")
 ✔ Navigated to url "file:///.../HTMLElements.html"
> highlight(radioButton('Yellow'))
 ✔ Highlighted the radio Button with label Yellow 
```
> The radio Button is not getting highlighted

**Version**
```
commit - e3952e957e4c49f248c26b1fd9ce7505cc8b57a8
```

**Comment by Debashis9012** [comment]
This issue has been verified and found fixed with the version 95d994b
